torch/lib/c10d/TCPStore.cpp

```diff
@@ -301,9 +301,16 @@ TCPStore::TCPStore(
   if (isServer_) {
     // Opening up the listening socket
     std::tie(masterListenSocket_, tcpStorePort_) = tcputil::listen(masterPort);
+    
+    std::string socketAddr = tcputil::getLocalSocketAddr(masterListenSocket_);
+    LOG(INFO) << "Master listen socket addr: " << socketAddr;
```

Comment:
Can we demote this to a `LOG(DEBUG)`? It might be too verbose to have this show up in INFO logs.